src/site/_includes/codelab.njk

```diff
@@ -36,14 +36,12 @@ show_banner: false
       {% if related_post %}
         {% ArticleNavigation {
           back: '/' + related_post,
-          backLabel: 'Return to article',
-          lang: lang
+          backLabel: 'Return to article'
         } %}
       {% else %}
         {% ArticleNavigation {
           back: '/' + path.slug,
```

Comment:
are you still thinking of getting rid of `path` in a future PR?

Reply:
yes, but it still requires some untangling (e.g. postToCollections collection relies on it) and bigger files reshuffling, and you said you wanted small PRs:)